src/Console/Application.php

```diff
@@ -119,24 +117,13 @@ public function run(InputInterface $input = null, OutputInterface $output = null
         return parent::run($input, $output);
     }
 
-    public function doRun(InputInterface $input, OutputInterface $output)
+    public function doRun(InputInterface $input, OutputInterface $output): int
     {
-        $output->writeln(self::LOGO);
+        $output->writeln([self::LOGO, $this->getLongVersion()]);
 
         return parent::doRun($input, $output);
     }
 
-    public function getLongVersion()
```

Comment:
Why did we remove it?

I guess now it returns `Infection - PHP Mutation Testing Framework dev-master@a1eb3df1012a2c86c0c32de9503e03140b1b7e9d` instead of `Infection - PHP Mutation Testing Framework dev-master`?

Reply:
It was removed because it wasn't used.